DemoApp/Sources/Views/CallView/DemoCallingTopView.swift

```diff
@@ -40,6 +42,14 @@ struct DemoCallingTopView: View {
             }
 
             Spacer()
+            
+            if GIDSignIn.sharedInstance.currentUser != nil {
```

Comment:
Should we make that available only on pre-release builds?

Reply:
I think we should have it on the release as well. The sign in fails if you don't have getstream account. Would probably encourage employees to use it more, while there's no impact for other users.

Reply:
On Android production builds (in Google Play) we don't have a manul sign-in flow yet, but maybe we could add a button to login with Google (which would log out the random user and log you in with Google).

Reply:
I think it's important that it's not easily visible for a non-Stream user (i.e. it's not on the main page). That's why we put it in the login, which I think it's a good place.
Other option is to invoke it with some hidden gesture maybe?

Reply:
The login page isn't visible or reachable in Release builds. That's why i think that we should be hiding it for non-internal builds otherwise we need to move it to a screen that's reachable or make the login screen reachable